packages/cypress/src/utils/versions.ts

```diff
@@ -1,3 +1,4 @@
 export const nxVersion = '*';
 export const cypressVersion = '^8.3.0';
+export const cypressRequestVersion = '2.88.7'; // TODO: remove this as soon as the `har-validator` is fixed on cypress side
```

Comment:
@meeroslav could we add the Cypress repo issue link to the TODO so it's easier to follow up?